editors/code/src/main.ts

```diff
@@ -93,6 +93,12 @@ export async function activate(context: vscode.ExtensionContext) {
         activateHighlighting(ctx);
     }
     activateInlayHints(ctx);
+
+    vscode.workspace.onDidChangeConfiguration(
+        _ => ctx?.client?.sendNotification('workspace/didChangeConfiguration', { settings: "" }),
```

Comment:
Hm, is there perhaps some way to make the client library we use do this automatically?